model_api/python/model_api/models/result_types.py

```diff
@@ -0,0 +1,247 @@
+#
```

Comment:
Something for future, but maybe we can split this under `types/anomaly.py, types/classification.py`

Reply:
Yeah, that's a possible option